pkg/controller/certificaterequests/venafi/venafi.go

```diff
@@ -162,7 +163,20 @@ func (v *Venafi) Sign(ctx context.Context, cr *cmapi.CertificateRequest, issuerO
 
 	log.V(logf.DebugLevel).Info("certificate issued")
 
+	// Assume the last certificate is the root CA
+	var (
+		block, lastBlock *pem.Block
+		remainingBytes   = certPem
+	)
+	for {
+		block, remainingBytes = pem.Decode(remainingBytes)
+		if block == nil {
+			break
+		}
+		lastBlock = block
+	}
 	return &issuerpkg.IssueResponse{
 		Certificate: certPem,
+		CA:          pem.EncodeToMemory(lastBlock),
```

Comment:
This is what had been confusing me....I'd been adding `lastBlock.Bytes` to the CA field, which is the DER content of the PEM block.